site2/website/release-notes.md

```diff
@@ -1,6 +1,123 @@
+   Maven module [#3309](https://github.com/apache/pulsar/pull/3309)
+ * Improve Javadocs [#](https://github.com/apache/pulsar/pull/3592)
+ * Support specifying multiple hosts in pulsar service url and web url [#3249](https://github.com/apache/pulsar/pull/3249)  
+ * Automatically discover when partitions on a topic are increased [#3513](https://github.com/apache/pulsar/pull/3513)
+ * Added `Client.getPartitionsForTopic()` [#2972](https://github.com/apache/pulsar/pull/2972)
+   ([Javadoc](https://pulsar.apache.org/api/client/org/apache/pulsar/client/api/PulsarClient.html#getPartitionsForTopic-java.lang.String-))
+ * Added `Consumer.pauseMessageListener()` and `Consumer.resumeMessageListener()` [#2961](https://github.com/apache/pulsar/pull/2961)
+ * Removed shading relocations for Circe-checksum and lz4 libraries, to ensure native libraries
+   are correctly loaded when using shaded client lib. [#2191](https://github.com/apache/pulsar/pull/2191)
+
+#### Python client
+ * Fixed `Message.properties()` [#3595](https://github.com/apache/pulsar/pull/3595)
+
+#### Go client
+ * Added `Producer.flush()` to flush all outstanding messages [#3469](https://github.com/apache/pulsar/pull/3469)
+ * Support `Consumer.Seek()` [#3478](https://github.com/apache/pulsar/pull/3478)
+ * Added `Message.Topic()` [#3346](https://github.com/apache/pulsar/pull/3346)
+ * Allow to specify `SubscriptionInitPos` option in `ConsumerOptions` [#3588](https://github.com/apache/pulsar/pull/3588)
+ * Added TLS hostname verification [#3580](https://github.com/apache/pulsar/pull/3580)
+ * Allow to link statically against `libpulsar.a`[#3488](https://github.com/apache/pulsar/pull/3488)
+ * Expose `Producer.LastSequenceID()` and `Message.SequenceID()` [#3416](https://github.com/apache/pulsar/pull/3416)
+
+#### C++ client
+ * Enable batching by default when using `sendAsync()` [#2949](https://github.com/apache/pulsar/pull/2949)
+ * Allow to specify schema info in Avro format [#3354](https://github.com/apache/pulsar/pull/3354)
+ * Added `Producer.flush()` to flush all outstanding messages [#3020](https://github.com/apache/pulsar/pull/3020)
+ * Added TLS hostname verification [#2475](https://github.com/apache/pulsar/pull/2475)
+ * Allow to specify `SubscriptionInitialPosition` [#](https://github.com/apache/pulsar/pull/3567)
+ * Added `Message.getTopicName()` [#3326](https://github.com/apache/pulsar/pull/3326)
+ * Added `Cosnsumer.receiveAsync()` [#3389](https://github.com/apache/pulsar/pull/3389)
+ * Build `libpulsar.a` with all required dependencies [#3488](https://github.com/apache/pulsar/pull/3488)
+ * Removed Boost from Pulsar API headers [#3374](https://github.com/apache/pulsar/pull/3374)
+
+#### Adaptors
+
+ * Kafka client wrapper, added support for explicit partitioning and custom partitioner [#3462](https://github.com/apache/pulsar/pull/3462)
+ * Support config `auto.offset.reset` to Pulsar KafkaConsumer [#3273](https://github.com/apache/pulsar/pull/3273)
```

Comment:
I think there are multiple updates on Pulsar+Flink Integrations as well.

Reply:
Sure, let me go through the list of these

Reply:
Added some of the items from flink adapter. PTAL if I've missed something